txtdirect.go

```diff
@@ -235,8 +239,13 @@ func query(zone string, ctx context.Context, c Config) ([]string, error) {
 
 // Redirect the request depending on the redirect record found
 func Redirect(w http.ResponseWriter, r *http.Request, c Config) error {
+	if strings.Contains(r.URL.Path, "/metrics") {
```

Comment:
It's not gonna work if we remove this part. Prometheus doesn't get updated and return anything.

Reply:
Wouldn't this essentially mean redirect including "/metrics" will never work? That feels too connected in terms of different components.

Reply:
Prometheus requests don't contain a host, we can add another check that sees if r.URL.Host is empty or not.

Reply:
The scraping server could use a host to access the endpoint so that could to be problematic too.
Sidenote: Hardcoding metrics here would run into issues, when we configure it differently in the config.